nodejs/week3/src/app.js

```diff
@@ -0,0 +1,35 @@
+import express from "express";
+import dotenv from "dotenv";
+import knexInstance from "./connection.js";
+
+dotenv.config();
+
+const app = express();
+const port = process.env.PORT || 3000;
+
+app.use(express.json());
+
+app.get("/api/contacts", async (req, res) => {
+  let query = knexInstance.select("*").from("contacts");
+
+  if ("sort" in req.query) {
+    const allowedColumns = ["id", "name", "email", "phone"];
+    const orderBy = req.query.sort?.toString();
+    console.log("SQL", query.toSQL().sql);
+    if (orderBy && allowedColumns.includes(orderBy)) {
+      query = query.orderBy(orderBy);
```

Comment:
Don't you want to return a response when the query string does not contain the allowed columns?